src/components/table/TableCellRenderer.tsx

```diff
@@ -74,34 +85,52 @@ export function CellTextRenderer({
 export function CellBooleanRenderer({
   id,
   value,
+  className,
 }: {
   id: string;
   value?: string | number | any[] | ReactNode;
+  className?: string;
 }): JSX.Element {
   return (
-    <TableCell id={id} align='left'>
+    <TableCell id={id} align='left' className={className || ''}>
```

Comment:
I don't think all this ors are necessary. Classname can be undefined

Reply:
I added them because we keep seeing undefined in the html markup when inspecting elements. This is just cleaner.

Reply:
Oh, ok!